torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -344,6 +388,24 @@ def __init__(self,
                            padding, dilation, False, _pair(0), groups, bias, padding_mode,
                            eps, momentum, freeze_bn, qconfig, dim=2)
 
+    def to_float(self):
```

Comment:
is there a way for this to call `_ConvBnNd.to_float` to reuse more code?

Reply:
oh sorry, I have implemented this in ConvBnNd, forgot to delete the extra code